packages/base/src/asset-registries/Illustrations.js

```diff
@@ -13,18 +14,58 @@ const registerIllustration = (name, { dialogSvg, sceneSvg, spotSvg, set, title,
 	});
 };
 
+const registerIllustrationLoader = async (illustrationName, loader) => {
+	loaders.set(illustrationName, loader);
+};
+
+const _loadIllustrationCollectionOnce = async illustrationName => {
```

Comment:
As the loaders are on illustration level, not on collection level, I think more accurate is to rename `_loadIllustrationCollectionOnce` to `_loadIllustrationOnce`

Reply:
Corrected, ty